app/src/main/java/protect/card_locker/AboutActivity.kt

```diff
@@ -2,119 +2,167 @@ package protect.card_locker
-import androidx.core.view.isVisible
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.ArrowBack
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.material3.TopAppBarDefaults
+import androidx.compose.ui.Modifier
 
 import com.google.android.material.dialog.MaterialAlertDialogBuilder
+import protect.card_locker.ui.CatimaAboutSection
 
-import protect.card_locker.databinding.AboutActivityBinding
 
-class AboutActivity : CatimaAppCompatActivity() {
+class AboutActivity : ComponentActivity() {
     private companion object {
         private const val TAG = "Catima"
     }
 
-    private lateinit var binding: AboutActivityBinding
     private lateinit var content: AboutContent
 
+    @OptIn(ExperimentalMaterial3Api::class)
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        binding = AboutActivityBinding.inflate(layoutInflater)
         content = AboutContent(this)
         title = content.pageTitle
-        setContentView(binding.root)
-        setSupportActionBar(binding.toolbar)
-        enableToolbarBackButton()
-
-        binding.apply {
-            creditsSub.text = content.copyrightShort
-            versionHistorySub.text = content.versionHistory
-
-            versionHistory.tag = "https://catima.app/changelog/"
-            translate.tag = "https://hosted.weblate.org/engage/catima/"
-            license.tag = "https://github.com/CatimaLoyalty/Android/blob/main/LICENSE"
-            repo.tag = "https://github.com/CatimaLoyalty/Android/"
-            privacy.tag = "https://catima.app/privacy-policy/"
-            reportError.tag = "https://github.com/CatimaLoyalty/Android/issues"
-            rate.tag = "https://play.google.com/store/apps/details?id=me.hackerchick.catima"
-            donate.tag = "https://catima.app/donate"
-
-            // Hide Google Play rate button if not on Google Play
-            rate.isVisible = BuildConfig.showRateOnGooglePlay
-            // Hide donate button on Google Play (Google Play doesn't allow donation links)
-            donate.isVisible = BuildConfig.showDonate
-        }
-
-        bindClickListeners()
-    }
 
-    override fun onOptionsItemSelected(item: MenuItem): Boolean {
-        return when (item.itemId) {
-            android.R.id.home -> {
-                finish()
-                true
+        setContent {
+            MaterialTheme {
+                Scaffold(
```

Comment:
I would extract the logic for the actual screen content in a separate function. That would make it easier to preview the UI and test it with fake data. [Example](https://gitlab.com/AuroraOSS/AuroraStore/-/blob/master/app/src/main/java/com/aurora/store/compose/ui/commons/BlacklistScreen.kt#L162), [Documentation](https://developer.android.com/develop/ui/compose/tooling/previews)